src/app/project/list/list-item-form/edit-list-item/edit-list-item.component.html

```diff
@@ -0,0 +1,45 @@
+<dsp-progress-indicator *ngIf="loading"></dsp-progress-indicator>
+
+<div *ngIf="!loading" class="form-content list-info">
+
+    <!-- list label -->
+    <dsp-string-literal-input
+        [placeholder]="'Child node label'"
+        [value]="labels"
+        (dataChanged)="handleData($event, 'labels')">
+    </dsp-string-literal-input>
+    <span class="invalid-form" *ngIf="formInvalidMessage">{{ formInvalidMessage }}</span>
+
+    <br><br>
+
+    <!-- list description / comment -->
+    <dsp-string-literal-input
+        [textarea]="true"
+        [placeholder]="'Child node description'"
+        [value]="comments"
+        (dataChanged)="handleData($event, 'comments')"
+        [language]="labels.length ? labels[0].language : 'en'">
+    </dsp-string-literal-input>
+
+    <div class="form-panel form-action">
+        <span>
+            <button
+                mat-button
+                type="button"
+                (click)="closeDialog.emit()">
+                {{ 'appLabels.form.action.cancel' | translate }}
+            </button>
+        </span>
+        <span class="fill-remaining-space"></span>
+        <span>
+            <button
+                mat-raised-button
+                type="submit"
+                color="primary"
+                [disabled]="saveButtonDisabled"
+                (click)="updateChildNode()">
+                Update
```

Comment:
you used `{{ 'appLabels.form.action.cancel' | translate }}` line 30, should use the format `{{ 'appLabels.form.action.update' | translate }}` here as well?! It exists in the `en.json` file

Reply:
This was André's code that I moved around. I changed the text to use the translate service for 'update' in 2c88e7e.